content/en/smart-contracts/fundamentals/aggregated-deal-making/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Aggregated deal-making"
+description: "Learn about Aggregated Deal-making on the Filecoin blockchain, where developers can combine small storage deals into larger, more attractive deals for storage providers."
+lead: "Filecoin is designed to store large data for extended periods. Small-scale data (<4 GiB) can be combined with other small deals into larger ones, either on-chain or off-chain. Smart contracts can handle programmatic data storing. This article explains the process, referring to small-scale data as _sub-piece data_."
+draft: false
+images: []
+type: docs
+menu:
+  smart-contracts:
+    parent: "smart-contracts-fundamentals"
+    identifier: "aggregators"
+weight: 192
+toc: true
+---
+
+Aggregation is the process of combining multiple packages of _sub-piece data_ into a single package. This large package is stored on the Filecoin network instead of multiple smaller packages. Aggregation can be done off-chain or on-chain.
+
+## Process
+
+The base interface for aggregation requires the following components:
+
+1. A client who has data to upload.
+1. An aggregator platform that clients can interact with to make a deal and retrieve proof of storage.
```

Comment:
The usage of the term "deal" is confusing here. Assuming that the data referred to by (1) is a subpiece, the client does not make a deal with the aggregator. The client uploads the subpiece to the aggregator who, at its own volition, will eventually pack it into an aggregated deal.